dlp-terraform/lambda/preprocess_lambda_function.py

```diff
@@ -0,0 +1,28 @@
+import json
+import pandas as pd
+import numpy as np
+import traceback
+
+def preprocess(data, code):
```

Comment:
i understand that you needed to expose the handler we wrote in the lambda in order to be terraform compatible. we should setup some special check that enforces certain people to review this handler if someone inadvertently changes this file. You can enforce such restriction using the `CODEOWNERS` file @thomaschin35